csv_datasets/ldbc/lists/edges-sf-003-full-parquet.txt

```diff
@@ -63,12 +63,12 @@ sf-003/parquet/15/part.0.parquet
 sf-003/parquet/15/part.1.parquet
 
 KATANA_DEFAULT_LABEL=KNOWS
-creationDate:IGNORE|:START_ID(Person)|:END_ID(Person)
+creationDate:DATETIME|:START_ID(Person)|:END_ID(Person)
 sf-003/parquet/16/part.0.parquet
 sf-003/parquet/16/part.1.parquet
 
 KATANA_DEFAULT_LABEL=
```

Comment:
why is this empty?

Reply:
Don't know but it doesn't matter if every file has a specific label. There should be no default/empty labels for LDBC.

Reply:
@gliga How do you mean empty?